dotCMS/src/integration-test/java/com/dotcms/enterprise/rules/RulesAPIImplIntegrationTest.java

```diff
@@ -82,12 +79,80 @@ public void shouldSaveCondition() throws DotSecurityException, DotDataException
         final Rule rule = new RuleDataGen().host(host).nextPersisted();
 
         final ConditionGroup conditionGroup = new ConditionGroupDataGen().rule(rule).next();
-        this.addPermission(role, host, true);
+        this.addRulesPermission(role, host, true);
 
         //Saving and testing GroupCondition
         rulesAPI.saveConditionGroup(conditionGroup, user, false);
 
         List<Rule> allRules = rulesAPI.getAllRules(user, false);
+        checkCondition(user, conditionGroup, allRules);
+    }
+
+    /**
+     * Method to Test: {@link RulesAPIImpl#saveConditionGroup(ConditionGroup, User, boolean)}
+     *                  and {@link RulesAPIImpl#saveCondition(Condition, User, boolean)}
+     * When:Try to create a Group Condition and a Condition in a page's rule with PuBLISH permission over the page
```

Comment:
javadoc says `Condition Group` and Condition` but code shows only `Condition Group`

Reply:
done https://github.com/dotCMS/core/pull/18201/commits/a3d5384f514af695b89fdbc549338def9a8cce2c#diff-9baded4fb0fb63e2648d9c9c4380ec1aR157